.babelrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
+			{
+				targets: {
+					node: 'current'
+				}
+			}
+		]
+	]
+};
```

Comment:
I know we're using tabs for the bash, but it looks like we're using spaces for JS. Maybe we should use spaces here.

Reply:
🛠 fixed it